dialects/freebsd/dlsof.h

```diff
@@ -545,7 +545,11 @@ struct	namecache {
 #   else
 	LIST_ENTRY(namecache) nc_src;   /* source vnode list */
 	TAILQ_ENTRY(namecache) nc_dst;  /* destination vnode list */
+#   if  __FreeBSD_version >= 1300000
```

Comment:
`1300105` is the `__FreeBSD_version` bump after 2b86f9d6d013a9ad8b1f8d03286018e785d5b3f6.